### DISCUSSION TITLE
Updating Not Working.

### DISCUSSION THREAD

**Discussion by Arinji2** [pocketbase/pocketbase#3645] [Q&A] [answered]
Hiya, so i just noticed that the update keeps giving a 404. 

My Request Log

`
 ID	druuwlegle6dkwe
Status	404
Method	PATCH
Auth	auth_record
URL	/api/collections/playlists/records/f2id0oacixiblvc
Referer	N/A
Remote IP	172.17.0.1
User IP	2401:4900:1c97:908e:9dc:7927:28af:d0f3
UserAgent	undici
Meta	
{
  "errorDetails": "sql: no rows in result set",
  "errorMessage": "The requested resource wasn't found."
}
Created	
2023-10-30
02:51:11 UTC
`

The Playlist
![image](https://github.com/pocketbase/pocketbase/assets/87296817/f980d30b-39aa-4d0a-90a7-6c8ac3542dd2)

The Code
` pb.authStore.save(token);
    await pb.collection("users").authRefresh();

    const record = await pb.collection("playlists").update("f2id0oacixiblvc", {
      name: "test",
      public: true,
      image: null,
    });`

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that the update request satisfies the Update API rule of your "playlists" collection.

**Reply by Arinji2** [reply]
Yup works, I had my perms to update wrong.. I thought invalid perms would be a diff error, thanks doe.
